### DISCUSSION TITLE
fail to import txt file into ebsd by wizard

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#920] [Ask Anything] [answered]
Hello,
    I was trying to import ebsd data in txt format into mtex but failed. 
    My txt file contains 3 Euler angles, phase and x/y spatial coordinates as columns.
    I appreciate the help if any.
![image](https://user-images.githubusercontent.com/69452222/112949062-62cc0c80-916b-11eb-99f4-10fc380c9468.png)

**Comment by kilir** [comment]
Hi,
could you be a bit more precise what you mean by failed? How did you try  to load it?
Have a look at ` help loadEBSD_generic`.
Cheers,
Rüdiger

**Reply by Brian-Liu-rx** [reply]
Hi, kilir，
    Thanks for your reply!
    This is the screenshot when I imported the txt file into ebsd. It seems like my txt file is not a correct one.
    I am going to help loadEBSD_generic now.

**Reply by Brian-Liu-rx** [reply]
![image](https://user-images.githubusercontent.com/69452222/112951773-51383400-916e-11eb-8f82-867acacdc97d.png)

**Reply by Brian-Liu-rx** [reply]
I import the txt file by import_wizard('EBSD')

**Reply by Brian-Liu-rx** [reply]
and please notice that the txt. file is not generated by any EBSD software. It is wrote by myself, because I will use the results (orientations, phase and spatial coordinates) calculated by cellular automaton as input in the future.
Thanks!

**Comment by ralfHielscher** [comment, marked as answer]
Easiest workaround would be to replace the phaseNames like Aluminium by some phase Id. 

Ralf.

**Reply by Brian-Liu-rx** [reply]
Hi, Ralf.
It works! This is wonderful!
Thanks a lot!
